resources/js/langman.js

```diff
@@ -80,9 +80,9 @@ new Vue({
          * Add a new translation key
          */
         addNewKey(key) {
```

Comment:
This method is only used to add a new translation key, adding a new language should be a separate action.